Make.inc

```diff
@@ -947,6 +957,14 @@ FC += -mcpu=$(MCPU)
 JULIA_CPU_TARGET ?= $(MCPU)
 endif
 
+# Set MTUNE-specific flags
+ifneq ($(MTUNE),)
+CC += -mtune=$(MTUNE)
+CXX += -mtune=$(MTUNE)
+FC += -mtune=$(MTUNE)
+JULIA_CPU_TARGET ?= $(MTUNE)
```

Comment:
This is the only thing I'm unsure about; can `JULIA_CPU_TARGET` take anything `MTUNE` can?  Also, if the user set `MARCH` but not `MTUNE`, should we set `JULIA_CPU_TARGET` to something?

Reply:
`MTUNE` behaves the same as `MCPU` so I assume so. And also maybe incorrectly if we set `MARCH` we just pass it to JULIA_CPU_TARGET.
I think they're all supported because it get's passed into a clang like command line.